server/controllers/socket.controller.ts

```diff
@@ -53,23 +77,7 @@ async function handleJoin(userId: string, data: JoinEvent): Promise<boolean> {
 		throw new Error(`Updating room failed for room code ${data.roomCode}.`);
```

Comment:
We need to decide if we want to throw errors for these or not.   We could move this into the sendRoomData function as well and eliminate duplicated code.

Reply:
I think moving it in the sendRoomData function is fine. Less dupe code the better.

Reply:
Seeing some type errors with this change.
![image](https://user-images.githubusercontent.com/92868767/182474969-42c21203-3a9a-4d7c-abb3-83dc458e2b51.png)

Reply:
fixed.  thanks.